lib/svg-sprite/shape.js

```diff
@@ -432,9 +432,9 @@ SVGShape.prototype.getDimensions = function() {
 /**
  * Set the dimensions of this shape
  *
- * @param {number} width        Width
- * @param {number} height       Height
- * @returns {SVGShape}          Self reference
+ * @param {number|string} width        Width
```

Comment:
We should either keep this changes in docs or remove parseFloat

Reply:
Not sure what you mean exactly? If it's inconsistent with the docs, it might be a bug or a leftover.